awscli/customizations/eks/update_kubeconfig.py

```diff
@@ -287,17 +287,22 @@ def get_user_entry(self):
         Return a user entry generated using
         the previously obtained description.
         """
+
+        region = self._get_cluster_description().get("arn").split(":")[3]
```

Comment:
Are we guaranteed to always have the arn available to us? In line below (294 in this diff), there is a `get()` method call with a default of `""` if `arn` is missing, which is why I ask.

Reply:
Yes, this is computed based off of the cluster name, and will always be present.